File: RE/tempCodeRunnerFile.c
#include <stdio.h>
int main(){
char b[]="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
char c;
int i=6;
for (int i=6;i<15;i++){
    c=b[i+10];
	b[i+10]=b[i];
	b[i++]=c;
}
printf("%s",b);
}

File: TOOLS/tempCodeRunnerFile.py
import base64
code = "gJ1BRjQie/FIWhEslq7GxbnL26M4+HXUtcpmVTKaydOP38of5v90ZSwrkYzCAuND"
flag = "GQTZlSqQXZ/ghxxwhju3hbuZ4wufWjujWrhYe7Rce7ju"
coder = str.maketrans(code, "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/")
print(flag.translate(coder))
print(base64.b64decode(flag.translate(coder)))